py_gtfs_rt_ingestion/ingest.py

```diff
@@ -47,16 +44,31 @@ def parseArgs(args) -> dict:
         help='lambda event json file')
 
     parser.add_argument(
-        '--output',
-        dest='output_dir',
+        '--export',
+        dest='export_dir',
+        type=str,
+        help='where to export to')
+
+    parser.add_argument(
+        '--archive',
+        dest='archive_dir',
         type=str,
-        required=True,
-        help='provide a directory to output')
+        help='where to archive ingested files to')
 
-    parsed_args = IngestArgs(**vars(parser.parse_args(args)))
+    parser.add_argument(
+        '--error',
+        dest='error_dir',
+        type=str,
+        help='where to move unconverted files to')
 
-    if parsed_args.output_dir is not None:
-        os.environ['OUTPUT_DIR'] = parsed_args.output_dir
+    parsed_args = parser.parse_args(args)
+
+    if parsed_args.export_dir is not None:
+        os.environ['EXPORT_BUCKET'] = parsed_args.export_dir
+    if parsed_args.archive_dir is not None:
+        os.environ['ARCHIVE_BUCKET'] = parsed_args.archive_dir
+    if parsed_args.error_dir is not None:
+        os.environ['ERROR_BUCKET'] = parsed_args.error_dir
```

Comment:
add new args to parse and set env vars correctly from them.

we can still run this script locally, but it will do all of the moving stuff around on s3.

Reply:
id like to see if there is a cool poetry way to set these env vars so we won't have to pass these bucket names in every time we want to run this script.